static/js/compose.js

```diff
@@ -267,6 +267,11 @@ exports.start = function (msg_type, opts) {
     // Show either stream/topic fields or "You and" field.
     show_box_for_msg_type(msg_type, opts);
 
+    // Set focus to "Topic" when narrowed to a stream+topic and "New topic" button clicked.
+    if (opts.trigger === "new topic button") {
+        show_box('stream', $("#subject"), opts);
+    }
```

Comment:
This will call `show_box` twice, once above and once here, which seems wrong -- maybe make this an if/else?